packages/@dcl/ecs/src/engine/component.ts

```diff
@@ -62,7 +62,7 @@ export type ComponentDefinition<
    * Transform.get(otherEntity) // throw an error!!
    * ```
    */
-  get(entity: Entity): DeepReadonly<ComponentType<T>>
+  get(entity: DeepReadonly<Entity> | Entity): DeepReadonly<ComponentType<T>>
```

Comment:
Why is this necessary? Entity should be a opaque identifier

Reply:
I can take a look to the DeepReadonly type, but without this, it happens the next:
```ts
const myTransform = Transform.get(entityA)
// typeof myTransform.parent => DeepReadonly<Entity>
const myParentComponent = MeshCollider.get(myTransform.parent) // mismatch types between Entity and DeepReadonly<Entity>
```

Reply:
Does it make sense to change `DeepReadonly<Entity> -> Entity` in the transform definition then?